make/autoconf/flags-cflags.m4

```diff
@@ -574,11 +574,11 @@ AC_DEFUN([FLAGS_SETUP_CFLAGS_HELPER],
 
   # CXXFLAGS C++ language level for all of JDK, including Hotspot.
   if test "x$TOOLCHAIN_TYPE" = xgcc || test "x$TOOLCHAIN_TYPE" = xclang || test "x$TOOLCHAIN_TYPE" = xxlc; then
-    LANGSTD_CXXFLAGS="-std=c++14"
+    LANGSTD_CXXFLAGS="-std=c++17"
```

Comment:
No, this change cannot be made yet.  As noted in the prior JEP discussion, we need to wait for the aix-ppc
port maintainers to upgrade the compiler they are using.  I happened to check last with them about their
progress.  While progress has been made, they are not yet ready to throw that switch.  Hoping to finish that
work sometime this fall.

Reply:
This is resolved. JDK23 doesn't support the old compiler any more. The only supported AIX compiler is C++17 capable.